.github/workflows/ci.yml

```diff
@@ -0,0 +1,203 @@
+name: PIC CI
+
+on:
+  push:
+    branches: 
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  clang-format-check:
+    runs-on: macos-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+      - name: Install clang-format
+        run: |
+          brew install clang-format@8
+      - name: Run clang format check
+        run: |
+          bash scripts/check-clang.sh
+  mac-os-catalina-build-gcc:
+    runs-on: macos-10.15
+    env:
+      CC: gcc-11
+      CXX: g++-11
+      AWS_KVS_LOG_LEVEL: 7
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+      - name: Build repository
+        run: |
+          mkdir build && cd build
+          sh -c 'cmake .. -DBUILD_TEST=TRUE;cmake .. -DBUILD_TEST=TRUE'
+          make
+          ./kvspic_test --gtest_filter="-TimerQueueFunctionalityTest.*:HeapPerfTest.*"  
+  mac-os-catalina-build-clang:
```

Comment:
Rename to `mac-os-build-clang` given we are using `macos-latest`

Reply:
Updated in the latest commit